staging/src/k8s.io/kube-gen/test/apis/testgroup/v1/zz_generated.deepcopy.go

```diff
@@ -0,0 +1,130 @@
+// +build !ignore_autogenerated
+
+/*
+Copyright 2017 The Kubernetes Authors.
```

Comment:
Yes, I rebased the latest branch of upstream.

I guess kube-gen is generated previously and rebase didn't remove it. I removed the kube-gen by hand and waiting for the results.